edgelet/iotedged/src/lib.rs

```diff
@@ -3,10 +3,10 @@
 #![deny(rust_2018_idioms, warnings)]
 #![deny(clippy::all, clippy::pedantic)]
 #![allow(
-    clippy::doc_markdown, // clippy want the "IoT" of "IoT Hub" in a code fence
-    clippy::module_name_repetitions,
-    clippy::shadow_unrelated,
-    clippy::use_self,
+clippy::doc_markdown, // clippy want the "IoT" of "IoT Hub" in a code fence
```

Comment:
Indentation is wrong here.

Reply:
Not a change I made, was there since earlier so gonna keep it as is.

Reply:
![image](https://user-images.githubusercontent.com/5431382/58840303-f7ba5900-8619-11e9-91ce-db11d086e6cd.png)

Reply:
(Are you using a code review tool that ignores whitespace diffs?)

Reply:
yep sure enough, codeflow doesn't show this as a diff. Will fix!

Reply:
IIRC there's a button that looks like two stacked tildes that toggles whitespace-only diffs.